pyqtgraph/__init__.py

```diff
@@ -14,6 +14,7 @@
 
 ## 'Qt' is a local module; it is intended mainly to cover up the differences
 ## between PyQt and PySide.
+from . import palette
```

Comment:
Nitpick:
This file might want to live in the (already existing) 'colors' folder?

Reply:
moved to colors, but we might want to think about making it a package and renaming the module to qdarkstylePalettes so palettes.py is available in the future for general use palettes